### PR TITLE
Moved: Tencent.EDULite version 2.2.1.38

### PR DESCRIPTION
**Pull request by SpecterShell**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/28770)

### CODE REVIEW

**Comment by BajajAditya123** [comment]
uhmm.....
why are you creating so much of pr
create but do not break the repo

File: manifests/t/Tencent/EDULite/1.0/Tencent.TXEDULite.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Tencent.TXEDULite
```
Comment (ItzLevvie) [inline manifests/t/Tencent/EDULite/1.0/Tencent.TXEDULite.installer.yaml:4]:
```suggestion
PackageIdentifier: Tencent.EDULite
```

File: manifests/t/Tencent/EDULite/1.0/Tencent.TXEDULite.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.0.0.schema.json
+
+PackageIdentifier: Tencent.TXEDULite
```
Comment (ItzLevvie) [inline manifests/t/Tencent/EDULite/1.0/Tencent.TXEDULite.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Tencent.EDULite
```

File: manifests/t/Tencent/EDULite/1.0/Tencent.TXEDULite.locale.zh-CN.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Tencent.TXEDULite
```
Comment (ItzLevvie) [inline manifests/t/Tencent/EDULite/1.0/Tencent.TXEDULite.locale.zh-CN.yaml:4]:
```suggestion
PackageIdentifier: Tencent.EDULite
```

File: manifests/t/Tencent/EDULite/1.0/Tencent.TXEDULite.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Tencent.TXEDULite
```
Comment (ItzLevvie) [inline manifests/t/Tencent/EDULite/1.0/Tencent.TXEDULite.yaml:4]:
```suggestion
PackageIdentifier: Tencent.EDULite
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Make sure to rename the files from `Tencent.TXEDULite` to `Tencent.EDULite`

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/28770#pullrequestreview-763708714

**Review by ItzLevvie — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**